configuration.nix

```diff
@@ -78,6 +78,15 @@
   #   "djbooth007" = 1000;
   # };
 
+  ### Hardware wallets
+  # Enable this module to allow using hardware wallets. See https://github.com/bitcoin-core/HWI
+  # for more information.
+  # Ledger must be initialized through the official ledger live app and the Bitcoin app must
+  # be installed and running on the device.
+  # services.hardware-wallets.ledger = true;
+  # Trezor can be initialized with the trezorctl command in nix-bitcoin.
```

Comment:
This line and line 84-85 could be replaced by appropriate entries in the usage.md like #63

Reply:
Imo for this PR it's fine. You can remove a part of that in #63 if you want.I'd like concise comments in configuration.nix - especially if it helps us - but in this case seems fine to move it to usage.md. Maybe in #63 you can remove these lines and append a line like: "More detailed information can be found in docs/usage.md"?